### ISSUE TITLE
Seperate changelog and documentation in Readme.

### ISSUE THREAD

**Issue by SeppPenner** [aluxnimm/outlookcaldavsynchronizer#254]
I would recommend to separate the changelog from the user documentation in the readme. Why? Because nobody wants to scroll down to the documentation while reading all the changes made from version 0.x to version 3.x. I would recommend to make a new file called `changelog.md` in the main folder or something and link it from the `README.md` file.

**Comment by aluxnimm** [comment]
Yes good point. Just need to change the update logic then as well since changes are grepped from readme atm.

You can read documentation also at

https://caldavsynchronizer.org/help/documentation/
